src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -26,25 +38,30 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
         location: ProgressLocation.Notification,
         title: localize('verifyingWorkspace', 'Verifying workspace...')
     };
-    await window.withProgress(progressOptions, async () => {
-        const folder = await tryGetWorkspaceFolder(context);
-        if (folder) {
-            await setWorkspaceContexts(context, folder);
-        } else {
-            await showNoWorkspacePrompt(context);
-        }
-    });
-
-    const swaNode: StaticWebAppTreeItem = await node.createChild(context);
-    void showSwaCreated(swaNode);
-
-    const environmentNode: EnvironmentTreeItem | undefined = <EnvironmentTreeItem | undefined>(await swaNode.loadAllChildren(context)).find(ti => {
-        return ti instanceof EnvironmentTreeItem && ti.label === productionEnvironmentName;
-    });
-    environmentNode && await ext.treeView.reveal(environmentNode, { expand: true });
-
-    void postCreateStaticWebApp(swaNode);
-    return swaNode;
+
+    try {
+        await window.withProgress(progressOptions, async () => {
```

Comment:
I have an idea - you know how quick picks are disabled with a "loading" indicator while loading the picks? I wonder if we can do a fake quick pick just to show progress during wizards. The main benefit is that the progress is shown in the top center where the rest of the wizard happens instead of at the bottom right (which I think is part of the confusion with "Verifying Workspace")

Reply:
Yeah, I like this idea a lot.

Reply:
Yeah this idea would help us in a lot of places. We need a way to show loading while we do async work in between wizard steps.

For example, we do async work to load the initial value of an InputBox in several places. This results in a long period of time where no UI is shown. 

I have been experimenting with adding an option to pass in `Promise<string | undefined>` to `showInputBox`, however I'm having trouble making the experience nice due to some limits of the InputBox API.

I think the "fake quick pick" idea Eric mentioned is a great idea.